edweiss-app/model/lectures/lectureDoc.ts

```diff
@@ -20,6 +20,7 @@ namespace LectureDisplay {
 
 	export type LectureEvents = "quiz";
 	export type AvailableLangs = "english" | "french" | "spanish" | "italian" | "german" | "brazilian" | "arabic" | "chinese" | "vietanames" | "hindi";
+	export type QuestionID = string & {};
```

Comment:
That's a great insight of interfacing this type. That's a great practice.

Reply:
I can't take the credit I followed Youssef's model ideas haha